projects/schematics/src/migrations/2_0/constructor-deprecations/data/cart-item.component.migration.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  CART_ITEM_COMPONENT,
+  FEATURE_CONFIG_SERVICE,
+  PROMOTION_SERVICE,
+  SPARTACUS_CORE,
+  SPARTACUS_STOREFRONTLIB,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+// projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.ts
+export const CART_ITEM_COMPONENT_MIGRATIONS: ConstructorDeprecation[] = [
```

Comment:
@znikola would you mind evaluating this file? I'm unsure if the different migration scripts are what we should end up with?